src/build_tools.md

```diff
@@ -171,14 +171,84 @@ fn main(){
 }
 ```
 
+[ex-cc-custom-defines]: #ex-cc-custom-defines
+<a name="ex-cc-custom-defines"></a>
+## Compile a C library while setting custom defines
+
+[![cc-badge]][cc] [![cat-development-tools-badge]][cat-development-tools]
+
+It is simple to build bundled C code with custom defines using [`cc::Build::define`][cc-build-define]. 
+It takes an [`Option`] value, so it is possible to create defines such as `#define FLAG`
+as well as `#define VALUE 123`. This example builds a bundled C file with dynamic defines set in 
+`build.rs` and prints `Welcome to foo - version 1.0.2` when run. Cargo sets some [environment 
```

Comment:
> `build.rs` and prints `Welcome to foo - version 1.0.2` 

Lets leave the backtick "`" emphasis for code. I would suggest **bold** emphasis here.